src/redux/auth/reducer.js

```diff
@@ -16,17 +18,29 @@ const loginReducer = (state = initState, action) => {
                 errorMessage: "",
```

Comment:
Not related to this PR but I think the ...state shouldnt be needed here since, you are manually setting all the properties of the state anyways. I have removed in my code https://github.com/VPPR/frontend/blob/12c52a8b142dcc04640b44ca752b93f776b7843c/src/redux/auth/reducer.js#L14

Reply:
hmmm, this seems ok

Reply:
what do u mean?

Reply:
as in it's OK to make this change?